parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -398,6 +413,92 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Add bridge pallets (GPA)
+/// Add GRANDPA bridge pallet to track Polkadot relay chain on Kusama BridgeHub
+pub type BridgeGrandpaPolkadotInstance = pallet_bridge_grandpa::Instance1;
+impl pallet_bridge_grandpa::Config<BridgeGrandpaPolkadotInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgedChain = bp_polkadot::Polkadot;
+	type MaxFreeMandatoryHeadersPerBlock = ConstU32<4>;
+	type HeadersToKeep = RelayChainHeadersToKeep;
+	type WeightInfo = weights::pallet_bridge_grandpa::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const RelayChainHeadersToKeep: u32 = 600;
+	pub const ParachainHeadsToKeep: u32 = 300;
+	/// Delay (in blocks) before registered relayer could get its stake back. It guarantees
+	/// that all pending delivery transactions are either dropped or mined and relayer is
+	/// slashed in case of misbehavior. So this value should be large enough (e.g. 24 hours).
+	pub const RelayerStakeLease: u32 = 7_200;
```

Comment:
How big of a change is it to go to 14,400? It will change to this once we have async backing, so we should either set it now (which will be 48 hours), or ensure that this value gets changed along with the upgrade to async backing. Or perhaps add `HOURS`, `DAYS`, etc. constants that are linked to the block time such that this gets updated automatically when the block time config changes.

Reply:
@svyatonik Slava, please, wdyt here?

Reply:
Changed to `parachains_common::DAYS` - hope it makes sense